src/assets/translations/en/main.json

```diff
@@ -600,8 +601,20 @@
         "exit": "Unstake",
         "removeLiquidityETH": "Remove liquidity"
       },
-      "thor": {
-        "transferOut": "Transfer out"
+      "thorchain": {
+        "deposit": "Deposit",
+        "depositRefund": "Deposit Refund",
+        "loanOpen": "Open Loan",
+        "loanOpenRefund": "Open Loan Refund",
+        "loanOut": "Loan Collateral Received",
+        "loanRepayment": "Repay Loan",
+        "loanRepaymentRefund": "Repay Loan Refund",
+        "out": "Outbound Received",
+        "swapOut": "Swap Received",
```

Comment:
Ditto WRT these if we want to use a consistent verbage, this would be "Receive Swap"

Reply:
https://github.com/shapeshift/web/pull/5986/commits/f58e07a45534dc01157b56082a72de7b4d8ded5f